docs/install/windows_setup.md

```diff
@@ -206,13 +207,28 @@ We have installed MXNet core library. Next, we will install MXNet interface pack
 - [Julia](#install-the-mxnet-package-for-julia)
 - **Scala** is not yet available for Windows
 
-## Install MXNet for Python
+## Install the MXNet Package for Python
 
 These steps are required after building from source. If you already installed MXNet by using pip, you do not need to do these steps to use MXNet with Python.
 
 1. Install ```Python``` using windows installer available [here](https://www.python.org/downloads/release/python-2712/).
 2. Install ```Numpy``` using windows installer available [here](https://scipy.org/index.html).
-3. Next, we install Python package interface for MXNet. You can find the Python interface package for [MXNet on GitHub](https://github.com/dmlc/mxnet/tree/master/python/mxnet).
+3. Start ```cmd``` and create a folder named ```common```(```mkdir C:\common```)
+4. Download the [mingw64_dll.zip](https://sourceforge.net/projects/openblas/files/v0.2.12/mingw64_dll.zip/download), unzip and copy three libraries (.dll files) that openblas.dll depends on to ```C:\common```.
+5. Copy the required .dll file to ```C:\common``` and make sure following libraries (.dll files) in the folder.
+```
+libgcc_s_seh-1.dll (in mingw64_dll)
+libgfortran-3.dll (in mingw64_dll)
+libquadmath-0.dll (in mingw64_dll)
+libopenblas.dll (in OpenBlas folder you download)
+opencv_world341.dll (in OpenCV folder you download)
+```
+6. Add ```C:\common``` to Environment Variables.
+ * Type ```control sysdm.cpl``` on ```cmp```
+ * Select the **Advanced tab** and click **Environment Variables**
+ * Double click the **Path** and click **New**
+ * Add ```C:\common``` and click OK
+7. Finally, we install Python package interface for MXNet. You can find the Python interface package for [MXNet on GitHub](https://github.com/dmlc/mxnet/tree/master/python/mxnet).
```

Comment:
I don't think you need this... just let the user continue on with running setup.py...

Reply:
How about putting these in a faq or reminder? I used OpenBLAS and was struggling to install python due to lack of those .dll files.

Reply:
I'm specifically talking about the link that goes to the old repo. Not only is that not the right link, I don't think users need step 7 at all.